Source/Scene/GroundPolylinePrimitive.js

```diff
@@ -0,0 +1,856 @@
+        '../Renderer/ShaderSource',
+        '../ThirdParty/when',
+        './BlendingState',
+        './CullFace',
+        './Material',
+        './PolylineColorAppearance',
+        './PolylineMaterialAppearance',
+        './Primitive',
+        './SceneMode'
+    ], function(
+        ApproximateTerrainHeights,
+        Check,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        GeometryInstance,
+        GeometryInstanceAttribute,
+        GroundPolylineGeometry,
+        isArray,
+        Matrix4,
+        PolylineShadowVolumeVS,
+        PolylineShadowVolumeFS,
+        PolylineShadowVolumeMorphVS,
+        PolylineShadowVolumeMorphFS,
+        DrawCommand,
+        Pass,
+        RenderState,
+        ShaderProgram,
+        ShaderSource,
+        when,
+        BlendingState,
+        CullFace,
+        Material,
+        PolylineColorAppearance,
+        PolylineMaterialAppearance,
+        Primitive,
+        SceneMode) {
+    'use strict';
+
+    /**
+     * A GroundPolylinePrimitive represents a polyline draped over the terrain in the {@link Scene}.
+     * <p>
+     *
+     * Only to be used with GeometryInstances containing GroundPolylineGeometries
+     *
+     * @param {Object} [options] Object with the following properties:
+     * @param {Array|GeometryInstance} [options.geometryInstances] GeometryInstances containing GroundPolylineGeometry
+     * @param {Appearance} [options.appearance] The Appearance used to render the polyline. Defaults to a white color {@link Material} on a {@link PolylineMaterialAppearance}.
+     * @param {Boolean} [options.show=true] Determines if this primitive will be shown.
+     * @param {Boolean} [options.vertexCacheOptimize=false] When <code>true</code>, geometry vertices are optimized for the pre and post-vertex-shader caches.
+     * @param {Boolean} [options.interleave=false] When <code>true</code>, geometry vertex attributes are interleaved, which can slightly improve rendering performance but increases load time.
+     * @param {Boolean} [options.compressVertices=true] When <code>true</code>, the geometry vertices are compressed, which will save memory.
+     * @param {Boolean} [options.releaseGeometryInstances=true] When <code>true</code>, the primitive does not keep a reference to the input <code>geometryInstances</code> to save memory.
+     * @param {Boolean} [options.allowPicking=true] When <code>true</code>, each geometry instance will only be pickable with {@link Scene#pick}.  When <code>false</code>, GPU memory is saved.
+     * @param {Boolean} [options.asynchronous=true] Determines if the primitive will be created asynchronously or block until ready. If false initializeTerrainHeights() must be called first.
+     * @param {ClassificationType} [options.classificationType=ClassificationType.TERRAIN] Determines whether terrain, 3D Tiles or both will be classified.
```

Comment:
Remove this, or leave it undocumented since it's unsupported for 3D Tiles.

Reply:
📋 🍝